tfjs-backend-webgpu/src/kernels/FromPixels.ts

```diff
@@ -60,14 +60,23 @@ export function fromPixels(args: {
         (pixels as HTMLVideoElement).videoHeight
       ] :
       [pixels.width, pixels.height];
-  const outShape = [height, width, numChannels];
+  const outputShape = [height, width, numChannels];
 
   const useImport = env().getBool('WEBGPU_USE_IMPORT') && isVideo;
   const isVideoOrImage = isVideo || isImage;
   if (isImageBitmap || isCanvas || isVideoOrImage) {
-    let externalImage;
+    let texture: GPUTexture|GPUExternalTexture;
+    let textureInfo: TextureInfo;
     if (useImport) {
-      externalImage = pixels as HTMLVideoElement;
+      const externalTextureDescriptor = {source: pixels as HTMLVideoElement};
+      texture = backend.device.importExternalTexture(externalTextureDescriptor);
+      textureInfo = {
+        width: pixels.width,
```

Comment:
Can you directly use the `width` `height` variables? `pixels.width` and `pixels.height` may not equal to `width` `height` in outputShape (see L57).

Reply:
done